docs/README.md

```diff
@@ -15,24 +15,11 @@ Head over to [Getting Started](./getting-started/README.md) to
 cluster](./getting-started/single-binary.md), and try [creating accounts and transfers using the
 CLI](./getting-started/cli-repl.md).
 
-## Designing for TigerBeetle
+## Building on TigerBeetle
```

Comment:
What about "Integrating with TigerBeetle" and `/integration/...` as the path? Or "Interfacing with TigerBeetle"?
I think "interfacing"/"integration" are just as -- or maybe more -- evocative as "building on".

...And also I like one-word path segments :stuck_out_tongue:

Reply:
I hear you on one-word path segments...

I don't love 'integrating with' because that makes me think more of an API service like Spotify, where they are the main thing and you're _integrating_ with them.

TigerBeetle is more _inside_ or _underneath_ your application.

Reply:
What about "Concepts"? Or "Semantics"? Though maybe those are too imprecise... :thinking: 

What do you think of "interfacing", just for the path name, if not also the sidebar label?

(It looks like DuckDB calls their equivalent section ["Connect"](https://duckdb.org/docs/connect) -- in the general sense of the word it is a good choice, but it has the other meaning of connecting via the network which is too narrow. I think "Interfacing" would be better.)

Another idea: "Applications", which has the (somewhat coinciding) double meaning of "user applications," and "how to apply tigerbeetle to your problem".

Reply:
If we're looking for one-word options, I'd lean more towards something like `build` and we could call the section "Build on TigerBeetle" (which might match the tense of "Deploy" better)

Reply:
:thinking: Could that be confused with compiling TB, or downloading builds?

Reply:
Yeah I thought of that. I think there's some slight possibility for confusion, but only slight. For compiling TB, I'd expect to find those instructions somewhere on the readme. And for downloading things, I'd expect that to be under "download".

Some other examples:
- Yugabyte uses Develop https://docs.yugabyte.com/preview/develop/
- Not exactly comparable but Supabase has Guides https://supabase.com/docs/guides/database/overview
- Redis has a bunch of sections but maybe the most comparable is their Manual https://redis.io/docs/manual/
- MongoDB also uses Manual (though that kind of includes all the docs) https://www.mongodb.com/docs/manual/data-modeling/

Reply:
Develop is nice, and contrasts nicely with deploy. But if you still like "Build on TigerBeetle" the best, then let's just change the url path to "build" and go with that.

Reply:
Develop works for me!